src/Apps/Order/Components/AddressVerificationFlow.tsx

```diff
@@ -203,6 +214,9 @@ const AddressVerificationFlow: React.FC<AddressVerificationFlowProps> = ({
     contextPageOwnerSlug,
   ])
 
+  if (verificationStatus === "VERIFIED_NO_CHANGE")
+    return <div data-testid="emptyAddressVerification"></div>
+
   if (addressOptions.length === 0) return null
```

Comment:
This got me to thinking about a bigger question: Do we need to do so much setup code in the [big useEffect here](https://github.com/artsy/force/pull/12689/files#diff-a65d5b9bf3f3ba0af862f0c53c8ea2e0be243adf95b86daacdb768a05e3f538aR158)? Or can we just load this all  outside of an effect on first render? I think this data should be available when the component is mounted.